### PR TITLE
[13.0][MIG] base_unece

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: base_unece/static/description/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils 0.14: http://docutils.sourceforge.net/" />
```
Comment (pedrobaeza) [inline base_unece/static/description/index.html]: Please update your local docutils to the same pinned version that is being used in OCA

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You should promote this module to development status "Production/Stable" and you as maintainer IMO.

Please also squash administrative some commits together, like stated in the migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by astirpe** [comment]
Yep :+1: done.

**Review by pedrobaeza — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM, functional review.

**Pull request merged**